docs/_templates/search.html

```diff
@@ -32,13 +50,92 @@
                 query,
                 {
                     hitsPerPage: 50,
+                    page: page,
                     facetFilters: [getVersionFacetFilter()]
                 }
             )
-                .then(({ hits }) => addResultsList(hits, query, searchResultsContainer))
+                .then(result => {
+                // Display pagination if more than 1 page returned
+                let nbPages = result["nbPages"]
+                if (nbPages > 1) {
+                    document.querySelector("#pagination").removeAttribute("hidden")
+                    displayPagination(page, nbPages)
+                }
+
+                // Display hits
+                let hits = result["hits"]
+                addResultsList(hits, query, searchResultsContainer)
+            })
                 .catch((error) => console.log(error))
         }
 
+        function displayPagination(page, totalPages) {
+            const paginationDiv = document.querySelector("#pagination")
+            const paginationList = paginationDiv.querySelector(".pagination-list")
+
+            // Hide previous/next button if showing first/last page
+            paginationDiv.querySelector(".pagination-previous").removeAttribute("hidden")
+            if (page == 0) {
+                paginationDiv.querySelector(".pagination-previous").setAttribute("hidden","hidden")
+            }
+            paginationDiv.querySelector(".pagination-next").removeAttribute("hidden")
+            if (page == totalPages - 1) {
+                paginationDiv.querySelector(".pagination-next").setAttribute("hidden","hidden")
+            }
+
+            // Display at most "toBeDisplayed" page links in the paginator
+            const toBeDisplayed = 10
+            let [start, end] = setStartEndForPaginator(page, totalPages, toBeDisplayed)
+
+            for (let i = start; i < end; i++) {
+                let newPaginationItem = document.createElement("li")
+                let newPaginationbutton = document.createElement("button")
+                let newPaginationLink = document.createElement("a")
+                newPaginationLink.classList.add("pagination-link")
+                newPaginationLink.innerHTML = i + 1
+                if (i == page) {
+                    newPaginationLink.setAttribute("aria-label", `page ${i+1}`)
+                    newPaginationLink.setAttribute("aria-current", `page`)
+                    newPaginationLink.classList.add("is-current")
+                } else {
+                    newPaginationLink.setAttribute("aria-label", `Go to page ${i+1}`)
+                }
```

Comment:
```suggestion
                newPaginationLink.setAttribute("aria-label", `page ${i+1}`)
                if (i == page) {
                    newPaginationLink.setAttribute("aria-current", "page")
                }
```

"go to" isn’t really needed – this is a link, so it’s implicit triggering the link will take the user to what the link label states.  And the `is-current` class shouldn’t be needed since we have an ARIA attribute just for that.